src/useSprings.js

```diff
@@ -25,9 +25,7 @@ export const useSprings = (length, props) => {
         const ctrl = new Ctrl()
         const newProps = isFn ? callProp(props, i, ctrl) : props[i]
         if (i === 0) ref = newProps.ref
-        ctrl.update(newProps)
-        if (!ref) ctrl.start()
```

Comment:
I think the `start` method should only be called within `useEffect`, to avoid starting the animation(s) before we know if the render was interrupted (which will be possible when Suspense is released). Am I misguided here?

Reply:
you may be right, putting it into useEffect should be fine

Reply:
Looks like we'll need to create the `Animated` objects eagerly since we're not calling `start` immediately. I guess we could call the `diff` method for all synchronous "updates". I'll test that out and report back with my findings.